stories/vrc-stories/Modal.js

```diff
@@ -74,6 +74,16 @@ storiesOf('Modal', module)
       />
     </Container>
   ))
+  .add('small - no content dark', () => (
+    <Container>
+      <Modal
+        title="Warning this is a Modal Title"
+        variant={Modal.VARIANT.SMALL}
+        headerTheme={Modal.HEADER_THEME.DARK}
```

Comment:
@aleonov-virtru would you add this as a Knob rather than a new story? I think it will be helpful to future developers.